callbacks/control_bar.py

```diff
@@ -97,3 +97,35 @@ def annotation_visibility(checked, store, figure, image_idx):
         patched_figure["layout"]["shapes"] = []
 
     return store, patched_figure
+
+
+clientside_callback(
+    """
+    function dash_filters_clientside(brightness, contrast) {
+    console.log(brightness, contrast)
+        js_path = "#image-viewer > div.js-plotly-plot > div > div > svg:nth-child(1) > g.cartesianlayer > g > g.plot > g"
+        changeFilters(js_path, brightness, contrast)
+        return ""
+    }
+    """,
+    Output("dummy-output", "children", allow_duplicate=True),
+    Input("figure-contrast", "value"),
```

Comment:
You've just flipped the order of the inputs here, so `brightness` and `contrast` are backwards